analyses/snv-callers/README.md

```diff
@@ -0,0 +1,211 @@
+# SNV caller comparison analysis
+
+This analysis evaluates [MAF files](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/)
+from different SNV callers and compares their output.
+The GDC has [good documentation on the fields](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/) contained in a standard MAF file.
+
+**Table of Contents**
+* [Individual caller evaluation](#individual-caller-evaluation)  
+  * [Base change analysis](#base-change-analysis)  
+  * [Variant allele fraction calculation](#variant-allele-fraction-calculation)  
+  * [Genomic regional analyses](#genomic-regional-analyses)  
+  * [Tumor mutation burden](#tumor-mutation-burden-calculation)
+  * [COSMIC mutation overlap](#cosmic-mutation-overlap)  
+* Comparison analysis - in development
+ * [Mutation IDs](#mutation-ids)
+* [Overall file structure](#overall-file-structure)
+* [Summary of functions](#summary-of-special-functions)
+## Individual Caller Evaluation
+
+The first step in this analysis is an individual evaluation of each MAF from
+each caller.
+This analysis prints out a report that is further subdivided by the WGS and WXS
+samples.
+
+Overall reports for each caller and strategy can be found:
+* `results/<caller_name>/<caller_name>_wgs_report.html`
+* `results/<caller_name>/<caller_name>_ws_report.html`
+
+### Base change analysis
+
+To evaluate base changes I summarized standard MAF fields as two new variables:
+The `base_change` variable that indicates the exact change in bases from
+concatenating `Reference_Allele`, `>`, and `Allele`.
+The `change` variable is made from the `base_change` variable but groups
+together deletions, insertions, and long (more than a SNV) as their own groups.
+
+*Output for this analysis*
+* `results/<caller_name>/<caller_name>_vaf.tsv`
+* `plots/<caller_name>/<caller_name>_<strategy>_base_change.png`
+
+### Variant Allele Fraction Calculation
+
+Calculate variant allele fraction (VAF) for each variant.
+```
+vaf = (t_alt_count) / (t_ref_count + t_alt_count)
+```
+ This is following the [code used in
+`maftools`](https://github.com/PoisonAlien/maftools/blob/1d0270e35c2e0f49309eba08b62343ac0db10560/R/plot_vaf.R#L39).
+The VAF calculations and other special variables are added to the MAF fields
+ and written to a TSV ending in `_vaf.tsv` in the caller's results folder.
+
+ *Output for this analysiss*
+ * `results/<caller_name>/<caller_name>_vaf.tsv`
+ * `plots/<caller_name>/<caller_name>_<strategy>_depth_vs_vaf.png`
+
+### Genomic Regional Analyses
+
+To analyze what genomic regions the variants are from, I used [Annotatr
+package](https://bioconductor.org/packages/release/bioc/vignettes/annotatr/inst/doc/annotatr-vignette.html) to obtain hg38 genome annotations. This Annotatr object is stored as an RDS file: `hg38_genomic_region_annotations.rds` in the `scratch` directory.
+Mutations are assigned all annotations that they overlap (using  
```

Comment:
All annotations? What do these include?

Reply:
Well, the reason I haven't listed them yet was because once I run it on the "real" datasets it may be we want to investigate other annotations. But for now, this is the annotations that are included, but you'll see this in a later script: 
```
    "hg38_basicgenes",
    "hg38_genes_intergenic",
    "hg38_genes_intronexonboundaries",
    "hg38_genes_exonintronboundaries",
    "hg38_genes_5UTRs",
    "hg38_genes_exons",
    "hg38_genes_introns",
    "hg38_genes_3UTRs"
```